Add DockerHub Descriptions for the gateway containers (#629)
ProjectLibertyLabs/gateway: Docker/account-service.overview.md

# Account Service Overview

> Account Service is a service enabling easy interaction with [DSNP](https://dsnp.org/) accounts on [Frequency](https://docs.frequency.xyz/). Accounts can be defined as a user's handle and MSA Id.

Reviewer: Can be is an odd phrasing. Trying out a different way to say it.
```suggestion
Account Service is a service enabling easy interaction with [DSNP](https://dsnp.org/) accounts on [Frequency](https://docs.frequency.xyz/). Accounts are identified by the Frequency MSA Id.
```

Author: Yeah, the summaries are copied directly from the `developer-docs/README.md`. We may want to keep those in sync, or a lot of that needs to go into the "Live Docs"
